docs/output.md

```diff
@@ -2,227 +2,9 @@
 
 ## Introduction
 
-This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
+This document describes the output produced by the pipeline.
 
-The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
+Please refer to the respective Output documentation:
```

Comment:
Could you add links here? sth like: 
-  [screening](https://github.com/nf-core/crisprseq/blob/dev/docs/output/screening.md)
-  [targeted](https://github.com/nf-core/crisprseq/blob/dev/docs/output/targeted.md)